core/src/main/scala/quasar/datasource/kafka/KafkaConsumer.scala

```diff
@@ -18,36 +18,42 @@ package quasar.datasource.kafka
 
 import slamdata.Predef._
 
-import org.apache.kafka.common.TopicPartition
+import org.apache.kafka.common.{PartitionInfo, TopicPartition}
+import org.slf4s.Logging
 
-import cats.Applicative
+import cats.Order
+import cats.data.NonEmptySet
 import cats.effect._
 import cats.implicits._
+import fs2.Stream
 import fs2.kafka.{CommittableConsumerRecord, ConsumerSettings, consumerResource}
-import fs2.{Stream, kafka}
 
-class KafkaConsumer[F[_]: Applicative: ConcurrentEffect: ContextShift: Timer, K, V](
+import scala.collection.immutable.SortedSet
+
+class KafkaConsumer[F[_]: ConcurrentEffect: ContextShift: Timer, K, V](
     settings: ConsumerSettings[F, K, V],
     decoder: RecordDecoder[F, K, V])
-    extends Consumer[F] {
+    extends Consumer[F] with Logging {
```

Comment:
🎉 I was going to make a comment about enabling logging and then got distracted :)